subcommand/inject-connect/command.go

```diff
@@ -129,16 +123,10 @@ func init() {
 
 func (c *Command) init() {
 	c.flagSet = flag.NewFlagSet("", flag.ContinueOnError)
-	c.flagSet.StringVar(&c.flagListen, "listen", ":8080", "Address to bind listener to.")
+	c.flagSet.IntVar(&c.flagListen, "listen", 8080, "Address to bind listener to.")
```

Comment:
I wonder if we should add it to changelog now as a breaking change. I think we might forget to mention it when we merge the feature branch.

Also, should it be port only?
```suggestion
	c.flagSet.IntVar(&c.flagListen, "listen", 8080, "Port to bind listener to.")
```

Reply:
Alternatively, to keep backward compatibility, we could keep our current set up and parse the string into host and port. Then set both when create new manager.

Reply:
I liked your second suggestion of maintaining backwards compatibility so i updated it to do the same and added some tests around the same!